docs/index.md

```diff
@@ -33,5 +33,4 @@ There are two categories of docs: [Guides](./guides/) and [API docs](./api/). Gu
 * [Removing Players](./guides/removing-players.md) - Helpful for using VideoJS in single page apps.
 
 ## API Docs
-- The most relevant API doc is the [player API doc](./api/vjs.Player.md).
```

Comment:
we should just link to the player doc: http://docs.videojs.com/docs/api/player.html